changelog.txt

```diff
@@ -1,3 +1,7 @@
+Version 5.0.0-preview
+==============
+- **Enhancements includes support for MSALv3 Cache (which is also or will soon be supported by other libraries such as Python and Java, more detail here https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/MSAL.NET-3-released) as well as enabling users to bring their own browser as part of the Auth Code retrieval.
```

Comment:
>s [](start = 24, length = 1)

i think it should be "Enhancements include" without the 's' but i'm not sure..have to say it out loud.

Reply:
you are right. no s